spacetemplate/assets/legacy.yaml

```diff
@@ -87,6 +87,121 @@ work_item_types:
   - *experienceID
   - *valuePropositionID
 
+work_item_boards:
+
+- id: "24181b5c-713f-4bef-a19f-45240875da92"
+  name: Scenarios Board
+  description: This is the default board config for the SDD template (Scenarios).
+  context: "679a563c-ac9b-4478-9f3e-4187f708dd30"
+  context_type: "TypeLevelContext"
+  columns:
```

Comment:
@michaelkleinhenz according to https://docs.google.com/presentation/d/18xKQmkf3UKlGoP7qLqUnrIYiflza674n_paUZXKuepA/edit?ts=5afda931#slide=id.g312a66d2fa_0_35 slide 19 there should be 5 columns. Why is the column for `Removed (meta state: mClosed` not added?

Reply:
@michaelkleinhenz the missing state is exactly the one I suggested to have in today's Grooming: Removed.

Reply:
Also, I see that `mClosed` state is not being used anywhere. We have added it only to `base.yaml` file.